packages/data-stores/src/onboard/constants.ts

```diff
@@ -9,9 +9,10 @@ export enum GoalKey {
 	Other = 'other',
 }
 
-export enum IntentKey {
+export enum SiteIntent {
 	Write = 'write',
 	Sell = 'sell',
 	Build = 'build',
 	DIFM = 'difm', // "Do It For Me"
+	WpAdmin = 'wpadmin',
```

Comment:
Nit: Not sure if this should be `WPAdmin`, `wpAdmin` or `WpAdmin`. All 3 use cases seem to exist across different codebases.